comp/otelcol/otlp/components/exporter/datadogexporter/factory.go

```diff
@@ -117,6 +118,11 @@ func CreateDefaultConfig() component.Config {
 	ddcfg := datadogconfig.CreateDefaultConfig().(*datadogconfig.Config)
 	ddcfg.Traces.TracesConfig.ComputeTopLevelBySpanKind = true
 	ddcfg.Logs.Endpoint = "https://agent-http-intake.logs.datadoghq.com"
+	ddcfg.QueueSettings.Batch = configoptional.Some(exporterhelper.BatchConfig{
+		Sizer:        exporterhelper.RequestSizerTypeItems,
+		FlushTimeout: 10 * time.Second,
+		MinSize:      int64(8192),
```

Comment:
Let's keep the sizes the same as OSS DD exporter: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/43419/files#r2419870122 MinSize 10 & MaxSize 100

Reply:
Just removed this section since it is set with the updated `CreateDefaultConfig()`